tools/json-api-client/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "@chainlink/json-api-client",
+  "private": true,
+  "version": "0.0.0",
+  "main": "./dist/src",
+  "types": "./dist/src",
+  "scripts": {
+    "build": "rm -rf dist && tsc",
```

This isn't windows friendly. Should we change it?

```suggestion
    "build": "rimraf dist && tsc",
```
Would need to add the `rimraf` dependency

---

This comes from https://github.com/smartcontractkit/chainlink/blob/develop/styleguide/package.json#L10

---

Lets just keep it for consistency then